src/app/app-platform/ContentAppPlatform.cpp

```diff
@@ -528,6 +528,12 @@ uint32_t ContentAppPlatform::GetPasscodeFromContentApp(uint16_t vendorId, uint16
         return 0;
     }
 
```

Comment:
Should you add the same update for the above on the line 504?

Reply:
Added!